content/v2.0/visualize-data/other-tools/grafana.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Use Grafana with InfluxDB v2.0
```

Comment:
I'd drop the v before 2.0 for consistency w other docs like "Get started with InfluxDB 2.0"